src/ctim/schemas/weakness.cljc

```diff
@@ -138,7 +140,7 @@
     (f/entry :effectiveness v/Effectiveness
              :description (str "summarizes how effective the mitigation may be "
                                "in preventing the weakness"))
-    (f/entry :effectiveness_notes c/MedString)))
+    (f/entry :effectiveness_notes c/Markdown)))
```

Comment:
```
(->> "/var/cwe/1000.xml"
       input-stream
       xml-input-stream->weakness-records
       (map :potential-mitigations)
       (mapcat identity)
       (map :mitigation)
       (map :effectiveness-notes)
       (remove nil?)
       (map type)
       set) => #{java.lang.String}
```
This change to `:effectiveness_notes` is not needed here, there are no cases in the production data where it uses anything other than a simple string.  The change to the `:description` IS needed, though!